### PR TITLE
Add select form filter

### PR DESCRIPTION
**Pull request by thayashi**
This commit enables to filter select menu, it'd be helpful to find the right item when there are so many list items.

### CODE REVIEW

**Comment by thayashi** [comment]
Example Image:
screen shot 2016-09-12 at 12 00 50 pm

File: app/js/views/formsEditors/selectEditor.js [hunk 1]
```diff
@@ -7,6 +7,24 @@ class SelectEditor extends Backbone.Form.editors.Select {
       super.setValue(value);
     }
   }
+
+  render() {
+    super.render();
+    const searchThreshold = 6;
+    const options = {
+      container: 'body',
+      width: 'auto'
```
Comment (stepek) [inline app/js/views/formsEditors/selectEditor.js]: Can you change ```width: 'auto'``` to ```width: '100%'``` 
With 100% it's looks better.

File: app/js/views/formsEditors/selectEditor.js [hunk 2]
```diff
@@ -7,6 +7,24 @@ class SelectEditor extends Backbone.Form.editors.Select {
...
+    };
+    const optionLength = Object.keys(this.schema.options).length;
+    if (optionLength >= searchThreshold) {
```
Comment (stepek) [inline app/js/views/formsEditors/selectEditor.js]: Please add new line before if statement.

File: app/js/views/formsEditors/selectEditor.js [hunk 3]
```diff
@@ -7,6 +7,24 @@ class SelectEditor extends Backbone.Form.editors.Select {
       super.setValue(value);
     }
   }
+
+  render() {
+    super.render();
+    const searchThreshold = 6;
```
Comment (stepek) [inline app/js/views/formsEditors/selectEditor.js]: Please add new line before const declaration.

**Comment by stepek** [comment]
LGTM

**Pull request merged**